app/models/user.model.js

```diff
@@ -21,6 +21,14 @@ var UserSchema = new Schema({
         type: String,
         required: true
     },
+    profilePicture: {
```

Comment:
I'm guessing `profilePicture` holds the url of the user's profile picture? In that case `profilePictureUrl` is probably a more accurate variable name.